fastFM/tests/test_ffm.py

```diff
@@ -38,5 +52,17 @@ def test_ffm2_predict_w0():
     y_pred = ffm2.ffm_predict(w0, w, V, X)
     assert_equal(y_pred, w0)
 
-if __name__ == '__main__':
-    test_ffm2_predict()
+# def test_ffm_fit():
+#     *_, y, X = get_test_problem()
+#     fm = als.FMRegression(n_iter=1000, init_stdev=0.1, rank=2, l2_reg_w=0.1, l2_reg_V=0.5)
+#     w0, w, V = ffm.ffm_als_fit(fm, X, y)
+#     y_pred = ffm.ffm_predict(w0, w, V, X)
+#
+#     assert_equal(np.round(y_pred), y)
+
+def test_ffm2_fit():
+    w0, w, V, y, X = get_another_test_problem()
+    rank = 2
+    w0, w, V = ffm2.ffm_als_fit(w0, w, V, X, y, rank)
```

Comment:
Feeding the solution as initial value into the solver is dangerous. The test will pass even if the solver doesn't change the parameters. It's important to check that the predictions become better then without calling fit. I would recommend initial values for w0=0, w[:] = 0 and V[:,:] = normal_distribution.